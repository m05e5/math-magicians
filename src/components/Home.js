/* eslint-disable linebreak-style */
import React from 'react';

const Home = () => (
  <div className="home">
    <h2>Wecome to our page!</h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Possimus voluptatibus aspernatur, molestias dicta enim fuga
      perspiciatis non ratione ullam, iste natus fugiat temporibus
      veniam impedit distinctio quis? Ab, commodi eligendi!
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Possimus voluptatibus aspernatur, molestias dicta enim fuga
      perspiciatis non ratione ullam, iste natus fugiat temporibus
      veniam impedit distinctio quis? Ab, commodi eligendi!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Possimus voluptatibus aspernatur, molestias dicta enim fuga
      perspiciatis non ratione ullam, iste natus fugiat temporibus
      veniam impedit distinctio quis? Ab, commodi eligendi!
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Possimus voluptatibus aspernatur, molestias dicta enim fuga
      perspiciatis non ratione ullam, iste natus fugiat temporibus
      veniam impedit distinctio quis? Ab, commodi eligendi!
    </p>
  </div>
);

export default Home;
