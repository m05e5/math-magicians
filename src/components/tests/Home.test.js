import React from 'react';

import Home from '../Home';

const ReactTestRenderer = require('react-test-renderer');

it('Creates a snapshot of the home page and it should match the previous one', () => {
  const tree = ReactTestRenderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
