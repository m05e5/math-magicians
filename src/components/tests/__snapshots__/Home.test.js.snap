// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a snapshot of the home page and it should match the previous one 1`] = `
<div
  className="home"
>
  <h2>
    Wecome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus voluptatibus aspernatur, molestias dicta enim fuga perspiciatis non ratione ullam, iste natus fugiat temporibus veniam impedit distinctio quis? Ab, commodi eligendi! Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus voluptatibus aspernatur, molestias dicta enim fuga perspiciatis non ratione ullam, iste natus fugiat temporibus veniam impedit distinctio quis? Ab, commodi eligendi!
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus voluptatibus aspernatur, molestias dicta enim fuga perspiciatis non ratione ullam, iste natus fugiat temporibus veniam impedit distinctio quis? Ab, commodi eligendi! Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus voluptatibus aspernatur, molestias dicta enim fuga perspiciatis non ratione ullam, iste natus fugiat temporibus veniam impedit distinctio quis? Ab, commodi eligendi!
  </p>
</div>
`;
